### PR TITLE
Added dynamic atlas and texture creation

### PR DESCRIPTION
**Pull request by CocosGames**

### CODE REVIEW

File: docs/zh/manuals/atlas.md [hunk 1]
```diff
@@ -114,3 +114,92 @@ Playback
...
   - `Loop Backward` 从最后一张图片到第一张图片循环播放.
   - `Loop Ping Pong` 从第一张图片播放到最后一张图片再反向循环播放.
+
+## 运行时纹理及图集建立
+
+自从 Defold 1.4.2 版本, 可以在运行时创建纹理和图集.
+
+### 在运行时创建纹理资源
+
+使用 (`resource.create_texture(path, params)`)[https://defold.com/ref/stable/resource/#resource.create_texture:path-table] 创建纹理资源:
```
Comment (britzl) [inline docs/zh/manuals/atlas.md:124]: I used the wrong markdown for this link. It should be []() and not ()[]. Could you please change it?

File: docs/zh/manuals/atlas.md [hunk 2]
```diff
@@ -114,3 +114,92 @@ Playback
...
+  local params = {
+    width  = 128,
+    height = 128,
+    type   = resource.TEXTURE_TYPE_2D,
+    format = resource.TEXTURE_FORMAT_RGBA,
+  }
+  local my_texture_id = resource.create_texture("/my_custom_texture.texturec", params)
+```
+
+纹理创建好之后就可以用 (`resource.set_texture(path, params, buffer)`)[https://defold.com/ref/stable/resource/#resource.set_texture:path-table-buffer] 设置纹理的像素:
```
Comment (britzl) [inline docs/zh/manuals/atlas.md:136]: Same

**Review by britzl — CHANGES_REQUESTED** [review]
Wrong link style in English version of manual

File: docs/zh/manuals/atlas.md (hunk 1 above)
Comment (CocosGames) [reply docs/zh/manuals/atlas.md:124]: ok, will fix it latter.

**Comment by CocosGames** [comment]
> Wrong link style in English version of manual

Fixed in new pr.

**Review by britzl — APPROVED** [review]

**Pull request merged**